src/transformers/models/visual_bert/modeling_visual_bert.py

```diff
@@ -1172,18 +901,45 @@ def forward(
-            Labels for computing the masked language modeling loss. Indices should be in ``[-100, 0, ...,
-            config.vocab_size]`` (see ``input_ids`` docstring) Tokens with indices set to ``-100`` are ignored
-            (masked), the loss is only computed for the tokens with labels in ``[0, ..., config.vocab_size]``
-        sentence_image_labels (``torch.LongTensor`` of shape ``(batch_size,)``, `optional`):
-            Labels for computing the sentence-image prediction (classification) loss. Input should be a sequence pair
-            (see :obj:`input_ids` docstring) Indices should be in ``[0, 1]``:
-
-            - 0 indicates sequence B is a matching pair of sequence A for the given image,
-            - 1 indicates sequence B is a random sequence w.r.t A for the given image.
-
-        Returns:
+            labels (:obj:`torch.LongTensor` of shape ``(batch_size, total_sequence_length)``, `optional`):
+                Labels for computing the masked language modeling loss. Indices should be in ``[-100, 0, ...,
+                config.vocab_size]`` (see ``input_ids`` docstring) Tokens with indices set to ``-100`` are ignored
+                (masked), the loss is only computed for the tokens with labels in ``[0, ..., config.vocab_size]``
+            sentence_image_labels (``torch.LongTensor`` of shape ``(batch_size,)``, `optional`):
+                Labels for computing the sentence-image prediction (classification) loss. Input should be a sequence
+                pair (see :obj:`input_ids` docstring) Indices should be in ``[0, 1]``:
+
+                - 0 indicates sequence B is a matching pair of sequence A for the given image,
+                - 1 indicates sequence B is a random sequence w.r.t A for the given image.
+
+            Returns:
+
+        Example::
+            >>> # Assumption: `get_visual_embeddings(image)` gets the visual embeddings of the image in the batch.
+            >>> from transformers import BertTokenizer, VisualBertForPreTraining
+
+            >>> tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
+            >>> model = VisualBertForPreTraining.from_pretrained('gchhablani/visualbert-vqa-coco-pre')
+
+            >>> inputs = tokenizer("The capital of France is {mask}.", return_tensors="pt")
+            >>> visual_embeds = get_visual_embeddings(image).unsqueeze(0)
+            >>> visual_token_type_ids = torch.ones(visual_embeds.shape[:-1], dtype=torch.long) #example
+            >>> visual_attention_mask = torch.ones(visual_embeds.shape[:-1], dtype=torch.float)
+
+            >>> inputs.update({{
+                "visual_embeds": visual_embeds,
+                "visual_token_type_ids": visual_token_type_ids,
+                "visual_attention_mask": visual_attention_mask
+            }})
+
+            >>> labels = tokenizer("The capital of France is Paris.", return_tensors="pt", padding="max_length", max_length=inputs["input_ids"].shape[-1]+visual_embeds.shape[-2])["input_ids"]
```

Comment:
This line will be too long, so you should compute the max_len on a separate line before

Reply:
Will do.